beets/importer.py

```diff
@@ -1385,6 +1412,11 @@ def apply_choice(session, task):
 
     task.add(session.lib)
 
+    # If ``set_fields`` is set, set those fields to the
+    # configured values.
+    if config['import']['set_fields']:
+        task.set_fields()
```

Comment:
Aha; got it! That makes sense. Let's leave it where it is—perhaps with a comment to clarify why it's there?—for this version at least.